beancount/__init__.py

```diff
@@ -7,9 +7,6 @@
 # Check the version requirements.
 import sys
 
-if (sys.version_info.major, sys.version_info.minor) < (3, 3):
```

Comment:
A check is nice, why remove it?

Reply:
I think it's more for development.
Can you keep the check and update the version number?

Reply:
I don't pip install anything, I mangle my `PYTHONPATH` like a dinosaur (it works well!)

Reply:
Yes